src/strings.c

```diff
@@ -1265,6 +1265,8 @@ string_from_blob(blob_T *blob, long *start_idx)
 
     if (str_ga.ga_data != NULL)
```

Comment:
The comment header for this function needs to be updated to reflect this change.